README.md

```diff
@@ -44,22 +46,23 @@ This section describes the quickest way to get started with ATM on a modern mach
    $ sudo apt install sqlite3
    ```
 
-   - for MySQL: 
+   - for MySQL:
    ```
    $ sudo apt install mysql-server mysql-client libmysqlclient-dev
    ```
 
-3. **Install python dependencies**
-  ATM is tested with Python 2.7+ and Python 3.5+.
+3. **Install python dependencies**.
+   This will also install [btb](https://github.com/hdi-project/btb), the core AutoML library in development under the HDI project, as an egg which will track changes to the git repository.
+   Here, usage of `virtualenv` is shown, but you can substitute `conda` or your preferred environment manager as well.
```

Comment:
Has this been tested with conda or anything else? I really don't know how conda works, so it might not be an issue.

Reply:
I use conda myself and was able to follow these instructions successfully, so I'd be comfortable at least recommending conda. I don't know if there are other environment managers besides these that are used.